src/openrct2/peep/Peep.cpp

```diff
@@ -2695,8 +2822,8 @@ static void peep_update_ride_sub_state_0(rct_peep * peep)
         }
     }
 
-    uint8 car_array_size = 0xFF;
-    uint8 car_array[255];
+    uint8 carArraySize = 0xFF;
+    uint8 carArray[255];
```

Comment:
Why does this declare a var `carArraySize`, but does not use it for the size of `carArray`?

Reply:
The array will be of variable size at this point we don't know. It is guaranteed to be at most 255 though

Reply:
But carArray is declared _immediately_ after carArraySize...

Reply:
Well either way it should be a vector.